helpdesk_mgmt_rating/models/helpdesk_ticket.py

```diff
@@ -82,12 +82,15 @@ def rating_get_ticket_id(self):
         return self.id
 
     def action_view_ticket_rating(self):
+        self.ensure_one()
         action = self.env["ir.actions.act_window"]._for_xml_id(
             "helpdesk_mgmt_rating.helpdesk_ticket_rating_action"
         )
         action["name"] = _("Ticket Rating")
         action_context = safe_eval(action["context"]) if action["context"] else {}
         action_context.update(self.env.context)
         action_context.pop("group_by", None)
+        if not action_context.get("id"):
+            action_context["id"] = self.id
```

Comment:
Why is this change necessary? In theory it should not be necessary.

Reply:
Otherwise it won't work, feel free to try it yourself

Reply:
I understand that, but I meant that this change should not be necessary because odoo core should take care of it; in any case, the important thing is that it works, so no problem.

Reply:
Ah yes, true. I also expected it to work out of the box, but alas.